pages/docs/tracing-features/masking.mdx

```diff
@@ -275,6 +275,167 @@ def llama_index_fn(question: str):
 
 </Tabs>
 
+## Example 1: Redacting Credit Card Numbers
```

Comment:
header structure of the page does not make sense now, can you group these as h2 examples, each example as h3 and add a h2 at the top for how-to

Reply:
![CleanShot 2025-01-17 at 11 37 57](https://github.com/user-attachments/assets/f0487836-edaa-44ac-997d-a46194443e59)

Reply:
when you add a description to the section of examples, you can add that we use the decorator here but this would apply to the other integrations as well